air/src/proof.rs

```diff
@@ -106,55 +113,33 @@ impl ProofOptions {
             fri_folding_factor,
             fri_max_remainder_size,
         );
-        Self { hasher, options }
+        Self { options, hash_fn }
     }
 
-    /// Creates a new preset instance with `[Blake3_192]` as hasher and 96-bits of security.
+    /// Creates a new preset instance of [ProofOptions] targeting 96-bit security level.
     pub fn with_96_bit_security() -> Self {
         let options = WinterProofOptions::new(27, 8, 16, FieldExtension::Quadratic, 8, 256);
         Self {
-            hasher: HashFunction::Blake3_192,
+            hash_fn: HashFunction::Blake3_192,
             options,
         }
     }
 
-    /// Creates a new preset instance with `[Blake3_256]` as hasher and 128-bits of security.
+    /// Creates a new preset instance of [ProofOptions] targeting 128-bit security level.
     pub fn with_128_bit_security() -> Self {
         let options = WinterProofOptions::new(27, 16, 21, FieldExtension::Cubic, 8, 256);
         Self {
-            hasher: HashFunction::Blake3_256,
-            options,
-        }
-    }
-
-    /// Creates a new preset instance with `[Rpo256]` as hasher and 128-bits of security.
-    pub fn with_rpo_128_bit_security() -> Self {
-        let options = WinterProofOptions::new(27, 16, 21, FieldExtension::Cubic, 8, 256);
-        Self {
-            hasher: HashFunction::Rpo256,
+            hash_fn: HashFunction::Blake3_256,
             options,
         }
     }
 
     // PUBLIC ACCESSORS
     // --------------------------------------------------------------------------------------------
 
-    /// Returns the representation of the underlying hasher.
-    pub const fn hasher(&self) -> HashFunction {
-        self.hasher
-    }
-
-    /// Returns the STARK protocol parameters.
-    pub const fn parameters(&self) -> &WinterProofOptions {
-        &self.options
-    }
-
-    // DESTRUCTOR
-    // --------------------------------------------------------------------------------------------
-
-    /// Extracts the underlying `[WinterProofOptions]`.
-    pub fn into_inner(self) -> WinterProofOptions {
```

Comment:
Should we remove the `into_inner`? It can be convenient for many cases. Also, I don't see now a method to access the winter options.

Reply:
I've added `impl From<ProofOptions> for WinterProofOptions` to convert `ProofOptions` into `WinterProofOptions`.